drools-ha/ha-core-infra/src/test/java/org/kie/hacep/LocalStorageKieSessionTest.java

```diff
@@ -127,7 +127,7 @@ public void deleteTest() throws ExecutionException, InterruptedException {
         assertEquals((Long) 0L, session.getFactCount().get());
     }
 
-    @Test(timeout = 10000)
+    @Test(timeout = 20000)
```

Comment:
here it timed out even for 20000ms https://rhba-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/KIE/job/master/job/upstream-downstream/job/droolsjbpm-build-bootstrap.downstream/87/testReport/org.kie.hacep/LocalStorageKieSessionTest/fireUntilHaltTest/

Reply:
Yep, I've added logs in the LocalSystem to understand where is the place of the stopped flow in this specific test

Reply:
So what do you want to do now? merge this and wait for failure on FDB?

Reply:
Jenkins execute full downstream build